contracts/HumanStandardToken.sol

```diff
@@ -42,16 +42,4 @@ contract HumanStandardToken is StandardToken {
         decimals = _decimalUnits;                            // Amount of decimals for display purposes
         symbol = _tokenSymbol;                               // Set the symbol for display purposes
     }
-
-    /* Approves and then calls the receiving contract */
```

Comment:
Was no one using this? Should it still be made available somewhere in this repo? (although I'm also not excited about having yet another inheritance step in here)

Reply:
We had a convo about this here and decided to remove it for now: https://consensys.slack.com/archives/C03DVPNTT/p1511107561000096